client/v2/autocli/testdata/msg-output.golden

```diff
@@ -1 +1 @@
-{"body":{"messages":[{"@type":"/cosmos.bank.v1beta1.MsgSend","from_address":"cosmos1y74p8wyy4enfhfn342njve6cjmj5c8dtl6emdk","to_address":"cosmos1y74p8wyy4enfhfn342njve6cjmj5c8dtl6emdk","amount":[{"denom":"foo","amount":"1"}]}],"memo":"","timeout_height":"0","unordered":false,"extension_options":[],"non_critical_extension_options":[]},"auth_info":{"signer_infos":[],"fee":{"amount":[],"gas_limit":"200000","payer":"","granter":""},"tip":null},"signatures":[]}
```

Comment:
why this change?

Reply:
Point 1, empty fields are omitted

Reply:
hmm, does this introduce malleability, cc @aaronc @kocubinski @testinginprod

Reply:
This is on the CLI side, I don't think it changes how we are verifying signatures, but yeah it's worth to look into it